src/raven.js

```diff
@@ -498,6 +496,41 @@ Raven.prototype = {
         }
     },
 
+    showReportDialog: function (options) {
+        if (!window.document) // doesn't work without a document (React native)
+            return;
+
+        options = options || {};
+
+        var lastEventId = options.eventId || this.lastEventId();
+        if (!lastEventId) {
+            throw new RavenConfigError('Missing eventId');
```

Comment:
Note that if you have Raven configured to accept an error, you will have set a DSN, and will thus never hit `Missing DSN`.